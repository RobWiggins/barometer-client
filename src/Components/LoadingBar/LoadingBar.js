import React from 'react';

export default class LoadingBar extends React.Component {
  render() {
    return (
      <div className="container">
        <div className="top"></div>
        <div className="loading">
          <i className="fa fa-spinner"></i>
        </div>
      </div>
    );
  }
}
