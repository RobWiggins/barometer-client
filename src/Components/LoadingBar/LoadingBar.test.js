import React from 'react';
import ReactDOM from 'react-dom';
import LoadingBar from './LoadingBar';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('testing LoadingBar Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LoadingBar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('testing LoadingBar component snapshot', () => {
    const wrapper = shallow(<LoadingBar />).find('.container');
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});