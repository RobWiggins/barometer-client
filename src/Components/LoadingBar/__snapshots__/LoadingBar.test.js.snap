// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing LoadingBar Component testing LoadingBar component snapshot 1`] = `
<div
  className="container"
>
  <div
    className="top"
  />
  <div
    className="loading"
  >
    <i
      className="fa fa-spinner"
    />
  </div>
</div>
`;
